pom.xml

```diff
@@ -1518,36 +1525,6 @@
         <artifactId>maven-enforcer-plugin</artifactId>
       </plugin>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
```

Comment:
Likewise, sure about this? where else do these source roots get specified for the benefit of the scala compiler? could be missing it but that's what I thought the purpose of this was.